test/sorted-array-index-search-test.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var assert = require('assert');
+var sortedArrayIndexSearch = require('../lib/sorted-array-index-search');
+
+var intComparator = function(a, b) {
+  return a - b;
+}
+var xComparator = function(a, b) {
+  return a.x - b.x;
+}
+
+describe('sorted-array-index-search', function() {
```

Comment:
Technically, this method is only for finding the insertion point of an element inside an array, not for resorting an entire array, so it doesn't have a concept of stableness. 

Also, because of the method (of using to comparator) to compare whether a changed element is still ordered compared to it's neighbours (in `simple-filtered-collection`) rather than, say, resorting on change, we know that the change events will be stable since we only resort if the comparator no longer matches. 

Aside: `Array.sort` is not stable in all Javascript engines...